index.html

```diff
@@ -21,6 +21,17 @@ <h2>
       </button>
     </form>
```

Comment:
When I try and use the form, it doesn't work (and sends an error saying: "please check that you have filled the form correctly". This is because the server is unable to interpret the form data that is sent because you have configured the express app to only accept JSON (ie, the "from" and "text" sent from your form has to be in JSON). This is the line that configures the server to only accept JSON: https://github.com/CodeYourFuture/node-challenge-chat-server/pull/138/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR6....... but, the form in index.html does not send JSON.... (This isn't your fault and there is no way around this in html, so we have two options. Either, we have to use a react app or some sort of JavaScript  event handler to get the form to send it in JSON, or we can change the line in server.js that specifies it can only accept JSON... I have left a message in your server.js file to show you how to do this. )